src/js/common/schemaform/save-load-actions.js

```diff
@@ -112,14 +113,19 @@ export function saveInProgressForm(formId, version, returnUrl, formData) {
     // If we don't have a userToken, fail safely
     if (!userToken) {
       dispatch(setSaveFormStatus(SAVE_STATUSES.noAuth)); // Shouldn't get here, but...
-      return;
+      if (__BUILDTYPE__ === 'development') {
+        return Promise.reject('no auth'); // Returning a rejected promise for testing purposes only
+      }
+
+      return; // eslint-disable-line consistent-return
```

Comment:
Adding these pointless `return`s because eslint gets angry when there's an inconsistent return, and I wanted to record _where_ the inconsistency was.